libs/core/src/components/sage-tooltip/sage-tooltip.scss

```diff
@@ -0,0 +1,172 @@
+:host {
+  /**
+
+    * @prop --background: Background of the tooltip
+    * @prop --color: Text color of the tooltip
+    * @prop --arrow-size: width and height of the arrow
+    * @prop --arrow-offset: arrow offset position
+
+    * @prop --overlay-border-radius: overlay border-radius
+    * @prop --overlay-font-size: overlay font-size
+    * @prop --overlay-line-height: overlay line-height
+    * @prop --overlay-padding: overlay padding
+    *
+    * @prop --arrow-pointing-down: shape for downward facing arrow
+    * @prop --arrow-pointing-to-the-right: shape for right facing arrow
+    * @prop --arrow-pointing-to-the-left: shape for left facing arrow
+    * @prop --arrow-pointing-up: shape for upward facing arrow
+    */
+
+  --background: #000;
```

Comment:
should this be #202327?

Reply:
Can we update this prop to include the word color v`--background-color` or are we expecting any type of background can be provided?